.doc_gen/templates/zonbook/sdk_chapter_template.xml

```diff
@@ -8,103 +8,151 @@
-            </info>
-            <xi:include href="file://AWSShared/code-samples/docs/{{.ExampleId}}_{{$.Language}}.xml"></xi:include>
-            {{- if .GuideTopic.Url}}
-            <para>For more information, see <ulink {{.GuideTopic.DocType}} url="{{.GuideTopic.Url}}">{{.GuideTopic.Text}}</ulink>.</para>
-            {{- end}}
-        </section>
-        {{- end}}
-        {{- if .CrossServiceExamples}}
-    </section>
-    {{- end}}
+    <variablelist role="termdef">
+        <para>The examples are divided into the following categories:</para>
+        <varlistentry>
+            <term>Single-service actions</term>
+            <listitem><para>Excerpts that illustrate how to write code to call individual service functions.</para></listitem>
+        </varlistentry>
+        <varlistentry>
+            <term>Single-service scenarios</term>
+            <listitem><para>Examples that show how to write code to accomplish a specific task with a single service.</para></listitem>
+        </varlistentry>
+        <varlistentry>
+            <term>Cross-service</term>
+            <listitem><para>Sample applications that combine multiple &AWS; services.</para></listitem>
+        </varlistentry>
+    </variablelist>
+    <para role="topiclist-abbrev">Examples</para>
     {{- if .ApiExamples}}
     <section id="{{$.LanguageSlug}}_code_examples_categorized" role="topic">
         <info>
-            <title id="{{$.LanguageSlug}}_code_examples_categorized.title">Examples using {{$.SdkEntity.Short}}</title>
-            <titleabbrev id="{{$.LanguageSlug}}_code_examples_categorized.titleabbrev">Examples</titleabbrev>
+            <title id="{{$.LanguageSlug}}_code_examples_categorized.title">Single-service examples using {{$.SdkEntity.Short}}</title>
+            <titleabbrev id="{{$.LanguageSlug}}_code_examples_categorized.titleabbrev">Single-service</titleabbrev>
             <abstract>
-                <para>The following code examples show how to use {{$.SdkEntity.Short}} with &AWS; services.</para>
+                <para>The following code examples show how to implement common scenarios and perform individual service actions by using
+                    {{$.SdkEntity.Short}} with &AWS; services.</para>
             </abstract>
         </info>
-        <para>The following code examples show how to use {{$.SdkEntity.Short}} with &AWS; services.</para>
-        <para role="topiclist-abbrev">Examples</para>
+        <para>The following code examples show how to perform actions and implement common scenarios by using
+            {{$.SdkEntity.Short}} with &AWS; services.
+            <emphasis>Actions</emphasis> are excerpts that illustrate how to write code to call individual service functions.
+            <emphasis>Scenarios</emphasis> demonstrate ways you can write code to accomplish a specific task with a single service.</para>
```

Comment:
<para>The following code examples show you how to perform actions and implement common scenarios by using the {{$.SdkEntity.Long}} with individual &AWS; services.
            <emphasis>_Actions_</emphasis> are code excerpts that show you how to call individual service functions.
            <emphasis>_Scenarios_</emphasis> are code examples that show you how to accomplish a specific task by calling multiple functions within the same service.</para>

Reply:
Done